changelogs/fragments/115_iosxr_move_common_functionality_to_util.yaml

```diff
@@ -0,0 +1,4 @@
+---
+bugfixes:
```

Comment:
This is probably a trivial change? If I understand correctly, this optimizes the code internally and is not something that affects the end user experience.

Reply:
The code changes look fine to me.

Reply:
okay will add changelog as trivial

Reply:
ack